src/apps/loyalty/client_side_analytics.ts

```diff
@@ -0,0 +1,3 @@
+import { initAnalytics } from "./analytics"
+
+initAnalytics()
```

Comment:
Case of premature optimization. My thought would be that `analytics.ts` could potentially contain _all_ analytics-related code, and an individual app could have its own `client_side_analytics` or whatever to bring in what it needs.

But since we just have loyalty for now...will simplify.